docs/topics/mta-web-assessing-apps.adoc

```diff
@@ -10,30 +10,34 @@ You can use the {ProductName} ({ProductShortName}) {WebName} to determine the ri
 
 [NOTE]
 ====
-The procedure described below uses a built-in questionnaire for assessing the risks involved in containerizing an application. By default, this procedure is required prior to reviewing the application. However, you might want to skip this step and use your own questionnaire instead. In that case, use the Skipping assessment option.
+The following procedure uses a built-in questionnaire for assessing the risks involved in containerizing an application. Before reviewing the application, you should perform this procedure by default. However, you might want to skip this step and use your own questionnaire instead. In that case, use the Skipping assessment option.
```

Comment:
The skipping assessment option is obsolete in MTA 7.0.0 .

Also, this section needs to be removed. 
6.1.2. Skipping assessment of applications

Reply:
Done